c_common/front_end_common_lib/src/recording.c

```diff
@@ -92,6 +92,12 @@ static read_request_packet_data *data_ptr;
 //! The time between buffer read messages
 #define MIN_TIME_BETWEEN_TRIGGERS 50
 
+//! n words outside struct per recording region used
+#define N_WORDS_USED_OUTSIDE_STRUCT_PER_REGION 2
```

Comment:
What are these mysterious words?

Reply:
these are actually the size of the recording_channel_t. I admit it should be a sizeof instead of a magic constant. Guess i didnt realise what it was, wehn we did this.

why they are at the bottom of the recording_data_t is beyond me, but i think tis to do with the buffer manager needing to know where to read.